docs/design.md

```diff
@@ -898,9 +903,11 @@ A +---------------------+
   +---------------------+
   | TLV area            |
   | +-----------------+ |
-  | | TLV area header | | <- struct image_tlv_info
+  | | TLV area header | | <- struct image_tlv_info (optional)
```

Comment:
might put also the protected TLV area header magic here

Reply:
Done.